src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -188,28 +194,31 @@ public override void OnActionExecuting(ActionExecutingContext context)
             base.OnActionExecuting(context);
         }
 
-        // TODO: FACAVAL
-        //[Route("admin/extensions/{name}/{*extra}")]
-        //[HttpGet]
-        //[HttpPost]
-        //[AllowAnonymous]
-        //public async Task<HttpResponseMessage> ExtensionWebHookHandler(string name, CancellationToken token)
-        //{
-        //    var provider = _scriptHostManager.BindingWebHookProvider;
-
-        //    var handler = provider.GetHandlerOrNull(name);
-        //    if (handler != null)
-        //    {
-        //        string keyName = WebJobsSdkExtensionHookProvider.GetKeyName(name);
-        //        if (!this.Request.HasAuthorizationLevel(AuthorizationLevel.System, keyName))
-        //        {
-        //            return new HttpResponseMessage(HttpStatusCode.Unauthorized);
-        //        }
-
-        //        return await handler.ConvertAsync(this.Request, token);
-        //    }
-
-        //    return new HttpResponseMessage(HttpStatusCode.NotFound);
-        //}
+        [HttpGet]
+        [HttpPost]
+        [Authorize(AuthenticationSchemes = AuthLevelAuthenticationDefaults.AuthenticationScheme)]
```

Comment:
I'm not sure what this attribute is saying - the AuthenticationScheme is AuthenticationScheme? Perhaps its just a naming issue.

Reply:
The attribute, in this case here, is specifying the authentication schemes expected/supported by this action. The naming follows the convention used by other authentication handlers.